commands/getGod.js

```diff
@@ -0,0 +1,106 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { createCanvas, loadImage } = require('canvas');
+const { MessageAttachment } = require('discord.js');
+const { makeRequest } = require('../create-signature');
+
+function splitString(str, limit = 11) {
+  if (str.indexOf(' ') == -1 || str.length < limit) return str;
+  const arr = str.split(' ');
+  let combine = '';
+  let count = 0;
+  for (let i = 0; i < arr.length; i++) {
+    if (i > 0 && arr[i].length > limit * 1.5) {
+      combine += '\n';
+    }
+    combine += arr[i] + ' ';
+    if (count < Math.floor(combine.length / limit)) {
+      //console.log(`Count: ${count}, Combine:\n${combine}`);
+      count = Math.floor(combine.length / limit);
+      combine += '\n';
+    }
+  }
+  return combine;
+}
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('getgod') // can only have command as lowercase letters
+    .setDescription('Replies with relevant god information.')
+    .addStringOption((option) =>
+      option
+        .setName('godname')
+        .setDescription('God name to search for')
+        .setRequired(true)
+    ),
+  async execute(interaction) {
+    await interaction.deferReply();
+    const godname = interaction.options.get('godname').value;
+
+    const godlist = await makeRequest('getgods', ['1']);
```

Comment:
probably specify what the 1 means because otherwise it's kinda like "why is there a 1 here"